torch/onnx/utils.py

```diff
@@ -633,8 +643,16 @@ def _node_getitem(self, k):
     sel = self.kindOf(k)
     return getattr(self, sel)(k)
 
+def register_custom_op_symbolic(symbolic_name, symbolic_fn):
+    ns, op_name = symbolic_name.split('::')
```

Comment:
Add assertion to check the format of the symbolic_name, should be `domain::op`.

Also, we need to check the domain doesn't conflict with existing domain